frontends/web/src/routes/account/send/send.tsx

```diff
@@ -613,8 +613,7 @@ class Send extends Component<Props, State> {
                   </Column>
                 </Grid>
               </ViewContent>
-              <ConfirmingWaitDialog
-                paired={paired}
```

Comment:
Let me know if we also wanna have the conditional render of `paired` somewhere in `ConfirmSend`. 🤔

Reply:
actually that is for BB01 😬 

Ideally find a solution that we don't have to test BB01 too much.. should we keep confirm-wait-dialog.tsx and conditionally show it for BB01?

or maybe split send into 2 different components like we do for receive https://github.com/BitBoxSwiss/bitbox-wallet-app/blob/bdf270c0bfc824f3dfda91176f76852eb4a524f7/frontends/web/src/routes/account/receive/index.tsx#L34-L36

wdyt?